### PR TITLE
add default search image and location 

### PR DESCRIPTION
**Pull request by bradfordcondon**
# Documentation

Issue #734

## Description

I've twice been unable to find the default search pages when reviewing PRs and answering issues, because I dont use them.  This PR describes where to find them in the default search page, and also adds an image because why not.

This is a minor addition to existing documentation and is therefore a 1-review PR.

screen shot 2018-11-13 at 9 32 20 am

### CODE REVIEW

File: docs/user_guide/searching/default_pages.rst [hunk 1]
```diff
@@ -3,3 +3,10 @@ Tripal Content-Specific Search Tools
 
 
 By default, Tripal will provide a search tool for every Tripal content type.  When a new content type is created, a new search tool is automatically created for that tool.
+
+These search tools will be available at ``/data_search/[content label]``, such as ``/data_search/mrna``.
```
Comment (almasaeed2010) [inline docs/user_guide/searching/default_pages.rst:7]: The search pages are also listed on the `structure > views` page if you have the `views_ui` module enabled.

Comment (almasaeed2010) [reply docs/user_guide/searching/default_pages.rst:7]: I think that might be worth mentioning?

Comment (bradfordcondon) [reply docs/user_guide/searching/default_pages.rst:7]: Ah thanks.  I actually dont have it enabled, so i guess thats why i couldnt find it in the menu.

**Comment by bradfordcondon** [comment]
thanks for the review and suggestion @almasaeed2010

**Pull request merged**